### ISSUE TITLE
Remove social plugins

### ISSUE THREAD

**Issue by monofox** [ZUKSev/ZUKS-Website#13]
cf. http://www.facebook.com/l.php?u=http%3A%2F%2Fwww.heise.de%2Fct%2Fartikel%2F2-Klicks-fuer-mehr-Datenschutz-1333879.html&h=BAQFtWWwC

**Comment by Eldorado234** [comment]
Just use linked icons.

**Comment by monofox** [comment]
Or Shariff project here on github (https://github.com/heiseonline/shariff). This works now with just ONE click.

**Comment by Eldorado234** [comment]
I changed my mind concerning this topic. I think it's better, when the people first visit the social media page before subscribing to it. I didn't get the information out of piwik, how many people clicked the like buttons. Probably the rate is very low.

Yvonne promised, that she will create some social media icons that are conforming to our CI. 

Ok?

**Comment by monofox** [comment]
Yeah of course it is a problem to identify it via piwik. For this it would be better to track it with a redirect link through the own site (e.g. with Twitter: Click on icon -> piwik.org/twitter -> twitter.com/ZUKSde). On the piwik.org/twitter you could count the access / track it through piwik (also if i don't like it very much, but just that you know how you could resolve it).

I like your approach to use just linked icons!
